src/api/java/com/minecolonies/api/colony/requestsystem/requestable/IDeliverable.java

```diff
@@ -21,10 +20,17 @@
     /**
      * Method called to get the amount of items that need to be in the stack.
      *
-     * @return The amount of items that
+     * @return The amount of items.
      */
     int getCount();
 
+    /**
+     * Method called to get the minimum amount required to fulfill this request.
+     *
+     * @return the minimum amount.
+     */
+    int getMinimumCount();
```

Comment:
Please add a setter as well. Additionally use the given count as the default minimum count not 1.

If the builder then wants to override his special behaviour can then set the minimum count to what ever is handy.
Setting the default to one for every request type changes the behaviour.

Reply:
I wanted to avoid the previous behaviour we had with the setters all over the place. Thus, you initiate this with the value in the constructor (like all the other values) or you make a new one. No need for a setter here.